src/theme/searcher/searcher.js

```diff
@@ -468,9 +468,17 @@ window.search = window.search || {};
         showResults(true);
     }
 
-    fetch(path_to_root + 'searchindex.json')
-        .then(response => response.json())
-        .then(json => init(json))        
+    fetch(path_to_root + 'searchindex.js')
```

Comment:
Why keeping this part of the code instead of just using the content from the `.catch()` call? There is no point in parsing JS.